main.go

```diff
@@ -1,18 +1,54 @@
 package main
 
 import (
+	"encoding/json"
+	"fmt"
 	"os"
 
 	odin "github.com/dream11/odin/app"
 	"github.com/dream11/odin/internal/cli"
 	"github.com/dream11/odin/internal/ui"
+	"github.com/dream11/odin/pkg/request"
 )
 
 var logger ui.Logger
 
+func getLatestVersion() string {
+	reqUrl := "https://api.github.com/repos/dream11/odin/tags"
+	req := request.Request{
+		Method: "GET",
+		URL:    reqUrl,
+	}
+	res := req.Make()
+	if res.Error != nil {
+		logger.Debug("Error making http req to fetch latest version: " + res.Error.Error())
+		return ""
+	}
+	var jsonResponse []map[string]interface{}
+	err := json.Unmarshal(res.Body, &jsonResponse)
+	if err != nil {
+		logger.Debug("Unable to unmarshal latest version response : " + res.Error.Error())
+		return ""
+	}
+
+	// return the latest tag
+	return jsonResponse[0]["name"].(string)
+}
+
+func isLatestVersion(currentVersion string, latestVersion string) bool {
+	return currentVersion == latestVersion
+}
+
 func main() {
 	c := cli.Cli(odin.App.Name, odin.App.Version)
 	exitStatus, err := c.Run()
+
+	latestVersion := getLatestVersion()
+	if latestVersion != "" && !isLatestVersion(odin.App.Version, latestVersion) {
+		logger.Warn(fmt.Sprintf("You are using odin version %s; however, version %s is available", odin.App.Version, latestVersion))
```

Comment:
shouldn't this be INFO instead of WARN?

Reply:
This is kept as `WARN` log because, this message should be interpreted as warning to user. Since, usage of older version of client might not behave as expected.